servers/audio_server.cpp

```diff
@@ -271,26 +271,40 @@ void AudioServer::_driver_process(int p_frames, int32_t *p_buffer) {
 
 		//master master, send to output
 		int cs = master->channels.size();
+
+		// take away 1 from the stride as we are manually incrementing one for stereo
```

Comment:
Doesn't match the current comment style guidelines. Similar issue a few lines below.

Reply:
Done! The joke's on me for trusting an approved PR…

Reply:
We've been more lenient with style in the `3.x` branch, as most of the effort to improve comment style started for 4.0.